Dockerfile

```diff
@@ -23,17 +23,18 @@ RUN groupadd -r saleor && useradd -r -g saleor saleor
 RUN apt-get update \
   && apt-get install -y \
   libffi8 \
-  libgdk-pixbuf2.0-0 \
+  libgdk-pixbuf-xlib-2.0-0 \
```

Comment:
I'm not sure why this change is something we want, those are two different packages

Reply:
Nevermind. I see that there is a new `libgdk-pixbuf-2.0-0` instead of `libgdk-pixbuf2.0-0`